pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -372,4 +372,12 @@ public String getSocks5ProxyUsername() {
     public String getSocks5ProxyPassword() {
         return Objects.nonNull(socks5ProxyPassword) ? socks5ProxyPassword : System.getProperty("socks5Proxy.password");
     }
+
+    public long getInitialBackoffIntervalNanos() {
+        return initialBackoffIntervalNanos
+    }
+
+    public long getMaxBackoffIntervalNanos() {
+        return maxBackoffIntervalNanos;
+    }
```

Comment:
We do not need to add these methods. They are generated by lombok for the associated variables.
```suggestion
```